examples/gno.land/r/gov/dao/v3/memberstore/memberstore.gno

```diff
@@ -89,6 +91,70 @@ func init() {
 			return t3.BasePower
 		},
 	})
+
+	members.SetTier(T1)
+	members.SetMember(T1, std.Address("g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj"), &Member{InvitationPoints: 3}) // Jae
+	members.SetMember(T1, std.Address("g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"), &Member{InvitationPoints: 3}) // Manfred
+	members.SetMember(T1, std.Address("g1e6gxg5tvc55mwsn7t7dymmlasratv7mkv0rap2"), &Member{InvitationPoints: 3}) // Milos
+	members.SetMember(T1, std.Address("g1jazghxvvgz3egnr2fc8uf72z4g0l03596y9ls7"), &Member{InvitationPoints: 3}) // Nemanja
+	members.SetMember(T1, std.Address("g1qhskthp2uycmg4zsdc9squ2jds7yv3t0qyrlnp"), &Member{InvitationPoints: 3}) // Petar
+	members.SetMember(T1, std.Address("g18amm3fc00t43dcxsys6udug0czyvqt9e7p23rd"), &Member{InvitationPoints: 3}) // Marc
+	members.SetMember(T1, std.Address("g1dfr24yhk5ztwtqn2a36m8f6ud8cx5hww4dkjfl"), &Member{InvitationPoints: 3}) // Antonio
+	members.SetMember(T1, std.Address("g19p3yzr3cuhzqa02j0ce6kzvyjqfzwemw3vam0x"), &Member{InvitationPoints: 3}) // Guilhem
+	members.SetMember(T1, std.Address("g1mx4pum9976th863jgry4sdjzfwu03qan5w2v9j"), &Member{InvitationPoints: 3}) // Ray
+	members.SetMember(T1, std.Address("g127l4gkhk0emwsx5tmxe96sp86c05h8vg5tufzq"), &Member{InvitationPoints: 3}) // Maxwell
+	members.SetMember(T1, std.Address("g1cpx59z5r8vzeww2fm4ezpz7yvjs7kptywkm864"), &Member{InvitationPoints: 3}) // Morgan
```

Comment:
```suggestion
	members.SetMember(T1, std.Address("g1m0rgan0rla00ygmdmp55f5m0unvsvknluyg2a4"), &Member{InvitationPoints: 3}) // Morgan
```

:D

Reply:
Just curious: How many hours did you need to brute force the address? :D